projects/sport-data-valley/manual_seeds/physical_complaints_responses.rb

```diff
@@ -164,4 +164,4 @@ def create
 # rubocop:enable Metrics/AbcSize
 # rubocop:enable Metrics/MethodLength
 
-RheumaResponses.create
+PhysicalComplaintsResponses.create
```

Comment:
is this correct? I would also expect the name of the class to be changed in this file?

Reply:
Correct, nice one. I missed that

Reply:
Hmm let me remove this and make a new PR for it, as some required changes to fix the issue are missing